### PR TITLE
[JBWS-4273] removed profiles wfly2300 wfly2400 and removed unneeded g…

### PR DESCRIPTION
**Pull request by rsearls**
…roovy scripts

### CODE REVIEW

File: modules/testsuite/pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <surefire.memory.args>-Xmx640m</surefire.memory.args>
     <surefire.jdwp.args>-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5005</surefire.jdwp.args>
     <surefire.management.args>-Dcom.sun.management.jmxremote</surefire.management.args>
-    <additionalGroovyScriptSuffix />
+    <additionalGroovyScriptSuffix>-elytron</additionalGroovyScriptSuffix>
```
Comment (jbliznak) [inline modules/testsuite/pom.xml:22]: can't we just get rid of this suffix property by removing it from files too when there is only elytron now?

Comment (rsearls) [reply modules/testsuite/pom.xml]: Done

**Comment by rsearls** [comment]
@jimma where can I find the failed build?

**Comment by jimma** [comment]
@rsearls  Please find it from : https://github.com/jbossws/jbossws-cxf/actions

**Comment by rsearls** [comment]
@jimma 
This failure is a system network issue.
Error:    WSSecurityPolicyExamples21xTestCase.test2121:154 » WebService org.apache.cxf.s..
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid 
certification path to requested target

The failures in JMS related tests mean the jms*.groovy script is not correct.  These tests will be excluded until that can be addressed.

**Comment by jimma** [comment]
> This failure is a system network issue.
Error: WSSecurityPolicyExamples21xTestCase.test2121:154 » WebService org.apache.cxf.s..
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid
certification path to requested target

@rsearls  It seems still the issue in standalone.xml. There is some security configuration is missing or can't be found.

**Pull request merged**